connector_easypost/data/product_packaging_template.xml

```diff
@@ -0,0 +1,364 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- Copyright 2016 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+
+<openerp>
```

Comment:
`<odoo noupdate="1">`

Although it would be nice if you actually split this data. Some things like the name, weight, etc. should be under a `<data noupdate="1">`, but things like the measurements should not. If you put the same record ID in two XML records, the other just updates the existing.